bonito/openvino/model.py

```diff
@@ -0,0 +1,93 @@
+import os
+import numpy as np
+import torch
+
+try:
+    from openvino.inference_engine import IECore, StatusCode
+    from .loader import torch2openvino
+except ImportError:
+    pass
+
+class OpenVINOModel:
+
+    def __init__(self, model, half, dirname):
+        self.model = model
+        self.alphabet = model.alphabet
+
+        model_name = model.config['model'] + ('_fp16' if half else '')
+        xml_path, bin_path = [os.path.join(dirname, model_name) + ext for ext in ['.xml', '.bin']]
+        self.ie = IECore()
+        if os.path.exists(xml_path) and os.path.exists(bin_path):
+            self.net = self.ie.read_network(xml_path, bin_path)
+        else:
+            self.net = torch2openvino(model)
```

Comment:
`.onnx` is not an option here? importer could help

Reply:
ONNX way will have lower efficiency (2D topology performs better).